Content.Shared/Chemistry/Reaction/ChemicalReactionSystem.cs

```diff
@@ -15,6 +15,11 @@ namespace Content.Shared.Chemistry.Reaction
 {
     public sealed class ChemicalReactionSystem : EntitySystem
     {
+        /// <summary>
+        /// Foam reaction protoId.
+        /// </summary>
+        public static readonly ProtoId<ReactionPrototype> FoamReaction = "Foam";
```

Comment:
I don't think this change is needed at this time.

Reply:
having Foam as magic string is bad <_< and i needed Foam... its worse to have it magic-stringed imo...